src/ashley/machina_extensions/forum_permission/migrations/0002_auto_20160607_0500.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.6 on 2016-06-07 03:00
+from __future__ import unicode_literals
```

Comment:
3 reasons here : 

1. I strictly followed the machina documentation [about overriding an application](https://django-machina.readthedocs.io/en/stable/customization/overriding_applications.html#import-the-application-models-if-needed)

2. I think this facilitates the comparison between the base application migrations and our own.

3. Existing installations of ashley have already played these 5 migrations, with the same name.
    If I introduce a new "squashed" migration file with another name, it will be played on these older installations and I don't want this. And if these 

I think that there are more drawbacks than advantages to introduce a squashed version of the machina migrations.

Reply:
:+1: 
Understood. It looked strange but the Machina situation explains it.